modules/sildisco/src/Auth/Process/AddIdp2NameId.php

```diff
@@ -124,7 +127,7 @@ public function process(array &$state): void
         $samlIDP = $state[self::IDP_KEY];
 
         if (empty($state[self::SP_NAMEID_ATTR])) {
-            \SimpleSAML\Logger::warning(
+            Logger::warning(
```

Comment:
"use ... as SS_Logger" would help distinguish the Logger, given the multitude of Logger's out there.